config/ac_basic_auth.go

```diff
@@ -7,16 +7,16 @@ import (
 	"github.com/avenga/couper/internal/seetie"
 )
 
-var _ Body = &BasicAuth{}
```

Comment:
Same as jwt, all acs must have this interface: https://github.com/avenga/couper/blob/fix-interfaces/config/configload/load.go#L203